conference.js

```diff
@@ -1030,6 +1033,15 @@ export default {
     },
 
     /**
+     * Returns the current local video track in use.
+     *
+     * @returns {JitsiLocalTrack}
+     */
+    getLocalVideoTrack() {
```

Comment:
I could access it directly but then I'd be accessing conference._room, which is supposed to be private and also has a note saying not to use it. The room variable within conference is private by scope.